diff --git a/source/main_menubar.cpp b/source/main_menubar.cpp
index 9a7cb72da..6a4423be7 100644
--- a/source/main_menubar.cpp
+++ b/source/main_menubar.cpp
@@ -852,8 +852,6 @@ void MainMenuBar::OnImportMonsterData(wxCommandEvent& WXUNUSED(event)) {
 	}
 }
 
-
-
 void MainMenuBar::OnImportMinimap(wxCommandEvent& WXUNUSED(event)) {
 	ASSERT(g_gui.IsEditorOpen());
 	// wxDialog* importmap = newd ImportMapWindow();
@@ -2260,7 +2258,7 @@ void MainMenuBar::LoadScriptsMenu() {
 		int scriptIndex = 0;
 		for (const auto& script : scripts) {
 			if (scriptIndex >= (SCRIPTS_LAST - SCRIPTS_FIRST)) {
-				break;  // Maximum scripts reached
+				break; // Maximum scripts reached
 			}
 
 			if (!script->isEnabled()) {
