diff --git a/source/definitions.h b/source/definitions.h
index c1c40eb62..e59cc8dac 100644
--- a/source/definitions.h
+++ b/source/definitions.h
@@ -30,7 +30,7 @@
 #define __LIVE_NET_VERSION__ 5
 
 #define MAKE_VERSION_ID(major, minor, subversion) \
-	((major) * 10000000 + (minor) * 100000 + (subversion) * 1000)
+	((major)*10000000 + (minor)*100000 + (subversion)*1000)
 
 #define __RME_VERSION_ID__ MAKE_VERSION_ID( \
 	__RME_VERSION_MAJOR__,                  \
