diff --git a/source/brush.h b/source/brush.h
index c2df4d2e2..9062223aa 100644
--- a/source/brush.h
+++ b/source/brush.h
@@ -82,7 +82,7 @@ class Brushes {
 	}
 
 protected:
-	typedef std::map<uint32_t, AutoBorder*> BorderMap;
+	// typedef std::map<uint32_t, AutoBorder*> BorderMap;
 	BrushMap brushes;
 	BorderMap borders;
 

From 6e12201365405e0af0c3425681378f0ef7d85d9d Mon Sep 17 00:00:00 2001
From: michyaraque <michyaraque@gmail.com>
Date: Fri, 23 Jan 2026 22:26:40 +0100
Subject: [PATCH 16/41] fix: remove unnecesary const

---
 source/complexitem.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

