diff --git a/CMakeLists.txt b/CMakeLists.txt
index 17a9ccbed..9bc85396c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,7 +15,7 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 if(MSVC)
-    set(CMAKE_CXX_FLAGS                "/W3 /D_CRT_SECURE_NO_WARNINGS")
+    set(CMAKE_CXX_FLAGS                "/W3 /EHsc /D_CRT_SECURE_NO_WARNINGS")
     set(CMAKE_CXX_FLAGS_DEBUG          "/Od /Zi /D__DEBUG__")
     set(CMAKE_CXX_FLAGS_MINSIZEREL     "/O1 /DNDEBUG")
     set(CMAKE_CXX_FLAGS_RELEASE        "/O2 /DNDEBUG")
@@ -108,5 +108,5 @@ add_executable(rme ${rme_H} ${rme_SRC})
 set_target_properties(rme PROPERTIES CXX_STANDARD 17)
 set_target_properties(rme PROPERTIES CXX_STANDARD_REQUIRED ON)
 
-include_directories(${CMAKE_SOURCE_DIR}/source ${Boost_INCLUDE_DIRS} ${LibArchive_INCLUDE_DIRS} ${OPENGL_INCLUDE_DIR} ${GLUT_INCLUDE_DIRS} ${ZLIB_INCLUDE_DIR} ${LUA_INCLUDE_DIR})
+include_directories(${CMAKE_SOURCE_DIR}/source ${Boost_INCLUDE_DIRS} ${LibArchive_INCLUDE_DIRS} ${OPENGL_INCLUDE_DIR} ${GLUT_INCLUDE_DIRS} ${ZLIB_INCLUDE_DIR} ${LUA_INCLUDE_DIR} ${LUA_INCLUDE_DIRS})
 target_link_libraries(rme ${wxWidgets_LIBRARIES} ${Boost_LIBRARIES} ${LibArchive_LIBRARIES} ${OPENGL_LIBRARIES} ${GLUT_LIBRARIES} ${ZLIB_LIBRARIES} ${LUA_LIBRARIES} cpr::cpr nlohmann_json::nlohmann_json sol2::sol2)

From 4a60d6f34476fb56fb3bc686d9df45f8a979c41a Mon Sep 17 00:00:00 2001
From: michyaraque <michyaraque@gmail.com>
Date: Sat, 24 Jan 2026 15:32:44 +0100
Subject: [PATCH 36/41] ci: support vcpkg cache

---
 .github/workflows/build-windows.yml | 11 +++++++++++
 1 file changed, 11 insertions(+)

