diff --git a/source/map_display.cpp b/source/map_display.cpp
index bdcf542b1..66ba45473 100644
--- a/source/map_display.cpp
+++ b/source/map_display.cpp
@@ -474,7 +474,7 @@ void MapCanvas::OnMouseMove(wxMouseEvent& event) {
 		}
 	}
 
-	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("onMouseMove", mouse_map_x, mouse_map_y, floor)) {
+	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("mouseMove", mouse_map_x, mouse_map_y, floor)) {
 		return;
 	}
 
@@ -591,7 +591,7 @@ void MapCanvas::OnMouseMove(wxMouseEvent& event) {
 void MapCanvas::OnMouseLeftRelease(wxMouseEvent& event) {
 	int map_x, map_y;
 	ScreenToMap(event.GetX(), event.GetY(), &map_x, &map_y);
-	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("onMouseRelease", map_x, map_y, floor, std::string("left"))) {
+	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("mouseRelease", map_x, map_y, floor, std::string("left"))) {
 		return;
 	}
 	OnMouseActionRelease(event);
@@ -600,7 +600,7 @@ void MapCanvas::OnMouseLeftRelease(wxMouseEvent& event) {
 void MapCanvas::OnMouseLeftClick(wxMouseEvent& event) {
 	int map_x, map_y;
 	ScreenToMap(event.GetX(), event.GetY(), &map_x, &map_y);
-	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("onMousePress", map_x, map_y, floor, std::string("left"))) {
+	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("mousePress", map_x, map_y, floor, std::string("left"))) {
 		return;
 	}
 	OnMouseActionClick(event);
@@ -662,7 +662,7 @@ void MapCanvas::OnMouseCenterRelease(wxMouseEvent& event) {
 void MapCanvas::OnMouseRightClick(wxMouseEvent& event) {
 	int map_x, map_y;
 	ScreenToMap(event.GetX(), event.GetY(), &map_x, &map_y);
-	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("onMousePress", map_x, map_y, floor, std::string("right"))) {
+	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("mousePress", map_x, map_y, floor, std::string("right"))) {
 		return;
 	}
 	if (g_settings.getInteger(Config::SWITCH_MOUSEBUTTONS)) {
@@ -675,7 +675,7 @@ void MapCanvas::OnMouseRightClick(wxMouseEvent& event) {
 void MapCanvas::OnMouseRightRelease(wxMouseEvent& event) {
 	int map_x, map_y;
 	ScreenToMap(event.GetX(), event.GetY(), &map_x, &map_y);
-	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("onMouseRelease", map_x, map_y, floor, std::string("right"))) {
+	if (g_luaScripts.isInitialized() && g_luaScripts.emitCancellable("mouseRelease", map_x, map_y, floor, std::string("right"))) {
 		return;
 	}
 	if (g_settings.getInteger(Config::SWITCH_MOUSEBUTTONS)) {

From 52a2483d89f4d812fe9e4fe90a91a107d26add7b Mon Sep 17 00:00:00 2001
From: michyaraque <michyaraque@gmail.com>
Date: Sat, 24 Jan 2026 14:44:01 +0100
Subject: [PATCH 33/41] build: update MSVC flags

---
 CMakeLists.txt | 47 ++++++++++++++++++-----------------------------
 1 file changed, 18 insertions(+), 29 deletions(-)

