diff --git a/source/map_display.h b/source/map_display.h
index 18dd6f075..ae72964d2 100644
--- a/source/map_display.h
+++ b/source/map_display.h
@@ -101,8 +101,12 @@ class MapCanvas : public wxGLCanvas {
 	}
 	virtual void GetScreenCenter(int* map_x, int* map_y);
 
-	virtual int GetClientWidth() const { return ClientMapWidth; }
-	virtual int GetClientHeight() const { return ClientMapHeight; }
+	virtual int GetClientWidth() const {
+		return ClientMapWidth;
+	}
+	virtual int GetClientHeight() const {
+		return ClientMapHeight;
+	}
 
 	void StartPasting();
 	void EndPasting();
@@ -113,7 +117,9 @@ class MapCanvas : public wxGLCanvas {
 	void UpdateZoomStatus();
 
 	void ChangeFloor(int new_floor);
-	void SetFloor(int new_floor) { floor = new_floor; }
+	void SetFloor(int new_floor) {
+		floor = new_floor;
+	}
 	int GetFloor() const {
 		return floor;
 	}
