diff --git a/source/action.cpp b/source/action.cpp
index a0e524f80..6d728e138 100644
--- a/source/action.cpp
+++ b/source/action.cpp
@@ -142,7 +142,6 @@ size_t Action::memsize() const {
 }
 
 void Action::commit(DirtyList* dirty_list) {
-	editor.selection.start(Selection::INTERNAL);
 	ChangeList::const_iterator it = changes.begin();
 	while (it != changes.end()) {
 		Change* c = *it;
@@ -281,7 +280,6 @@ void Action::commit(DirtyList* dirty_list) {
 		}
 		++it;
 	}
-	editor.selection.finish(Selection::INTERNAL);
 	commited = true;
 }
 
@@ -290,7 +288,6 @@ void Action::undo(DirtyList* dirty_list) {
 		return;
 	}
 
-	editor.selection.start(Selection::INTERNAL);
 	ChangeList::reverse_iterator it = changes.rbegin();
 
 	while (it != changes.rend()) {
@@ -410,7 +407,6 @@ void Action::undo(DirtyList* dirty_list) {
 		}
 		++it;
 	}
-	editor.selection.finish(Selection::INTERNAL);
 	commited = false;
 }
 
