diff --git a/source/json/json_spirit_writer.h b/source/json/json_spirit_writer.h
index 108a07f4d..0d8a35fa9 100644
--- a/source/json/json_spirit_writer.h
+++ b/source/json/json_spirit_writer.h
@@ -7,42 +7,41 @@
 // json spirit version 4.03
 
 #if defined(_MSC_VER) && (_MSC_VER >= 1020)
-# pragma once
+	#pragma once
 #endif
 
 #include "json_spirit_value.h"
 #include <iostream>
 
-namespace json_spirit
-{
-    // functions to convert JSON Values to text,
-    // the "formatted" versions add whitespace to format the output nicely
+namespace json_spirit {
+	// functions to convert JSON Values to text,
+	// the "formatted" versions add whitespace to format the output nicely
 
-    void         write          ( const Value& value, std::ostream&  os );
-    void         write_formatted( const Value& value, std::ostream&  os );
-    std::string  write          ( const Value& value );
-    std::string  write_formatted( const Value& value );
+	void write(const Value& value, std::ostream& os);
+	void write_formatted(const Value& value, std::ostream& os);
+	std::string write(const Value& value);
+	std::string write_formatted(const Value& value);
 
 #ifndef BOOST_NO_STD_WSTRING
 
-    void         write          ( const wValue& value, std::wostream& os );
-    void         write_formatted( const wValue& value, std::wostream& os );
-    std::wstring write          ( const wValue& value );
-    std::wstring write_formatted( const wValue& value );
+	void write(const wValue& value, std::wostream& os);
+	void write_formatted(const wValue& value, std::wostream& os);
+	std::wstring write(const wValue& value);
+	std::wstring write_formatted(const wValue& value);
 
 #endif
 
-    void         write          ( const mValue& value, std::ostream&  os );
-    void         write_formatted( const mValue& value, std::ostream&  os );
-    std::string  write          ( const mValue& value );
-    std::string  write_formatted( const mValue& value );
+	void write(const mValue& value, std::ostream& os);
+	void write_formatted(const mValue& value, std::ostream& os);
+	std::string write(const mValue& value);
+	std::string write_formatted(const mValue& value);
 
 #ifndef BOOST_NO_STD_WSTRING
 
-    void         write          ( const wmValue& value, std::wostream& os );
-    void         write_formatted( const wmValue& value, std::wostream& os );
-    std::wstring write          ( const wmValue& value );
-    std::wstring write_formatted( const wmValue& value );
+	void write(const wmValue& value, std::wostream& os);
+	void write_formatted(const wmValue& value, std::wostream& os);
+	std::wstring write(const wmValue& value);
+	std::wstring write_formatted(const wmValue& value);
 
 #endif
 }
