diff --git a/source/ext/pugiconfig.hpp b/source/ext/pugiconfig.hpp
index c8846bb7d..cdc9faef6 100644
--- a/source/ext/pugiconfig.hpp
+++ b/source/ext/pugiconfig.hpp
@@ -12,29 +12,29 @@
  */
 
 #ifndef HEADER_PUGICONFIG_HPP
-#define HEADER_PUGICONFIG_HPP
+	#define HEADER_PUGICONFIG_HPP
 
-// Uncomment this to enable wchar_t mode
-// #define PUGIXML_WCHAR_MODE
+	// Uncomment this to enable wchar_t mode
+	// #define PUGIXML_WCHAR_MODE
 
-// Uncomment this to disable XPath
-#define PUGIXML_NO_XPATH
+	// Uncomment this to disable XPath
+	#define PUGIXML_NO_XPATH
 
-// Uncomment this to disable STL
-// #define PUGIXML_NO_STL
+	// Uncomment this to disable STL
+	// #define PUGIXML_NO_STL
 
-// Uncomment this to disable exceptions
-// #define PUGIXML_NO_EXCEPTIONS
+	// Uncomment this to disable exceptions
+	// #define PUGIXML_NO_EXCEPTIONS
 
-// Set this to control attributes for public classes/functions, i.e.:
-// #define PUGIXML_API __declspec(dllexport) // to export all public symbols from DLL
-// #define PUGIXML_CLASS __declspec(dllimport) // to import all classes from DLL
-// #define PUGIXML_FUNCTION __fastcall // to set calling conventions to all public functions to fastcall
-// In absence of PUGIXML_CLASS/PUGIXML_FUNCTION definitions PUGIXML_API is used instead
+	// Set this to control attributes for public classes/functions, i.e.:
+	// #define PUGIXML_API __declspec(dllexport) // to export all public symbols from DLL
+	// #define PUGIXML_CLASS __declspec(dllimport) // to import all classes from DLL
+	// #define PUGIXML_FUNCTION __fastcall // to set calling conventions to all public functions to fastcall
+	// In absence of PUGIXML_CLASS/PUGIXML_FUNCTION definitions PUGIXML_API is used instead
 
-// Uncomment this to switch to header-only version
-#define PUGIXML_HEADER_ONLY
-#include "pugixml.cpp"
+	// Uncomment this to switch to header-only version
+	#define PUGIXML_HEADER_ONLY
+	#include "pugixml.cpp"
 
 // Tune these constants to adjust memory-related behavior
 // #define PUGIXML_MEMORY_PAGE_SIZE 32768
