diff --git a/source/gui_ids.h b/source/gui_ids.h
index e5761573a..ebedf517c 100644
--- a/source/gui_ids.h
+++ b/source/gui_ids.h
@@ -180,7 +180,17 @@ enum EditorActionID {
 	TOOLBAR_SIZES_4,
 	TOOLBAR_SIZES_5,
 	TOOLBAR_SIZES_6,
-	TOOLBAR_SIZES_7
+	TOOLBAR_SIZES_7,
+
+	// Script Manager Window IDs
+	SCRIPT_MANAGER_LIST,
+	SCRIPT_MANAGER_RELOAD,
+	SCRIPT_MANAGER_OPEN_FOLDER,
+	SCRIPT_MANAGER_CLEAR_CONSOLE,
+	SCRIPT_MANAGER_RUN_SCRIPT,
+
+	MAP_POPUP_MENU_SCRIPT_FIRST,
+	MAP_POPUP_MENU_SCRIPT_LAST = MAP_POPUP_MENU_SCRIPT_FIRST + 20
 };
 
 enum ToolBarID {
